local_planner/include/local_planner/local_planner_node.h

```diff
@@ -101,7 +101,7 @@ enum class MAV_STATE {
 
 class LocalPlannerNode {
  public:
-  LocalPlannerNode();
+  LocalPlannerNode(const bool tf_spin_thread);
```

Comment:
I think it would be cleaner to put ` = true` here to make it the default, then only override it in the test.